### PR TITLE
feat: (Replat 6378) Fix article regressions

### PR DESCRIPTION
**Pull request by jmars**
Screenshot_1558526348

### CODE REVIEW

**Comment by parkjh** [comment]
@jmars android test is failing

**Comment by nehasri89** [comment]
@jmars  can you pls put screenshots of the points you have fixed from REPLAT-6378

File: packages/article-in-depth/src/styles/shared.js [hunk 1]
```diff
@@ -64,8 +64,13 @@ const sharedStyles = {
   metaContainer: {
     alignItems: "center",
     marginHorizontal: spacing(2),
-    paddingBottom: 6,
-    paddingTop: spacing(2)
+    paddingBottom: spacing(2),
```
Comment (aronshamash) [inline packages/article-in-depth/src/styles/shared.js]: `paddingVertical`

File: packages/article-main-comment/src/styles/shared.js [hunk 2]
```diff
@@ -61,7 +61,14 @@ const sharedStyles = {
     flexWrap: "wrap"
   },
   metaContainer: {
-    alignItems: "center"
+    alignItems: "center",
+    width: "100%",
+    paddingBottom: spacing(2),
```
Comment (aronshamash) [inline packages/article-main-comment/src/styles/shared.js]: `paddingVertical`

**Comment by tuncaulubilge** [comment]
Looks like we need to use fixed versions for babel libs @aronshamash @jmars

**Pull request merged**